CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- The formatter now tries to keep a function body and its arguments on a single
+  line by first trying to split only its last argument on multiple lines
```

Comment:
Could you put this in a formatter section please?

And revert the other changes to this file.

Reply:
Done 👍